main.go

```diff
@@ -102,6 +102,7 @@ func main() {
 		FQDNTemplate:                   cfg.FQDNTemplate,
 		CombineFQDNAndAnnotation:       cfg.CombineFQDNAndAnnotation,
 		IgnoreHostnameAnnotation:       cfg.IgnoreHostnameAnnotation,
+		IgnoreTLSSpec:                  cfg.IgnoreTLSSpec,
```

Comment:
If we are to accept this, it should be more indicative of what it does
```suggestion
		IgnoreIngressTLSSpec:                  cfg.IgnoreIngressTLSSpec,
```

Reply:
you are right it is more clear. 
Done

Thanks for your review